provider/cmd/pulumi-resource-cloud-toolkit-aws/landingZone/organizationArgs.ts

```diff
@@ -0,0 +1,119 @@
+import * as aws from "@pulumi/aws";
+
+export interface OrganizationArgs {
+  /**
+   * The organization ID to import the Organization in the stack. If not set a new AWS Organization will be created. Defaults to undefined.
+   */
+  organizationId?: string;
+
+  /**
+   * The Organization policies to be applied.
+   */
+  policies?: OrganizationPoliciesArgs;
+
+  /**
+   * The list of AWS Account to be configured in the Organization.
+   */
+  accounts?: OrganizationAccountArgs[];
+}
+
+export interface OrganizationPoliciesArgs {
+  /**
+   * Deny IAM Account to leave the organization. Enabled by default.
+   */
+  denyLeaveOrganization?: OrganizationPolicyArgs;
+}
+
+export interface OrganizationPolicyArgs {
+  /**
+   * Enable the policy/
+   */
+  enabled?: boolean;
+
+  /**
+   * Import the policy with the given id
+   */
+  policyId?: string;
+}
+
+export interface OrganizationAccountArgs {
+  /**
+   * The name of the IAM Account.
+   */
+  name: string;
+
+  /**
+   * The email associated to the IAM Account.
+   */
+  email: string;
+
+  /**
+   * Admin role for the IAM Account.
+   */
+  adminRoleName?: string;
+
+  /*
+   *  The Organizational Unit to be used for the account.
+   */
+  ou?: string;
+
+  /**
+   * The AWS Account ID to be used to import the Account in the Organization. If not set, a new AWS Account will be created.
+   */
+  accountId?: string;
+
+  /**
+   * The parentId of the imported account.
+   */
+  parentId?: string;
+}
+
+export const defaultOrganizationAccount = {
+  adminRoleName: "superadmin",
+};
+
+export interface PolicyData {
```

Comment:
These interfaces are missing a docstring.

Reply:
This is an internal interface that is not exposed in the schema.